ls1intum/artemis-ios
# `General`: Release 1.4.0

## New
- Number of channel members shown in title bar
- Support for iOS 18 with Dark Mode app icon and floating tab bar on iPadOS 18
- Easier formatting of text when composing messages
- Profile Pictures in channel member list
- FAQ Tab
- Support for displaying images in chats

## Fixed
- Inconsistent titles for exercise and lecture count
- Formatting issues in Conversation Info when names contained non ascii-characters
- Message actions may have offset touch targets
- Message Reaction View may disappear delayed

## Testing
### How to install
1. <a href="https://testflight.apple.com/join/canwoVCk">Join Artemis TestFlight Beta: testflight.apple.com/join/canwoVCk</a>
2. Download & Install Latest Build of Version 1.4

### What to test
- Verify that all described bugs have been fixed
- Ensure that formatting text works
- Check whether profile pictures are shown in channel member list and that the correct actions are shown when tapping on a user
- Ensure that the buttons when long pressing a message all work
- Make sure sending and replying to messages etc still works

You can use a test server by entering its url on the login screen under "Select university">"Custom instance".
